keentools/geotracker/settings.py

```diff
@@ -659,17 +727,17 @@ def reload_mask_2d(self) -> None:
         _log.output(f'mask mode: {mask_source}')
         if mask_source == 'MASK_2D':
             _log.output(f'RELOAD 2D MASK: {geotracker.mask_2d}')
-            mask.image = get_background_image(geotracker.camobj, index=1)
+            mask.image = get_background_image_strict(geotracker.camobj, index=1)
             mask.inverted = geotracker.mask_2d_inverted
             mask.mask_threshold = geotracker.mask_2d_threshold
-            mask.channel = int(geotracker.mask_2d_channel[1])
+            mask.channel = geotracker.get_mask_2d_channel_value()
         elif mask_source == 'COMP_MASK':
             mask_image = geotracker.update_compositing_mask()
             _log.output('RELOAD 2D COMP_MASK')
             mask.image = mask_image
             mask.inverted = geotracker.compositing_mask_inverted
             mask.mask_threshold = geotracker.compositing_mask_threshold
-            mask.channel = 4  # RGB
+            mask.channel = 7  # RGB
```

Comment:
What does this value mean?

Reply:
7 = RGB bitmask (without Alpha). Comment was added